packages/tasit-contracts/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "tasit-contracts",
+  "version": "0.0.1",
+  "description": "",
+  "keywords": [
+    "ethereum",
+    "dapp",
+    "truffle",
+    "ganache",
+    "solidity"
+  ],
+  "author": "Tasit Labs",
+  "homepage": "https://github.com/tasitlabs/TasitSDK#readme",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tasitlabs/tasitsdk.git"
+  },
+  "scripts": {
+    "ganache:start": "npx ganache-cli -m 'beach swap combine paper music cook electric bullet trust actress liquid asthma' > /dev/null 2>&1 &",
+    "ganache:stop": "pkill -f ganache-cli",
+    "compile": "npx truffle compile > /dev/null 2>&1 &",
```

Comment:
So are we expecting that users usually run the test suite from the root of this project, which means that truffle can compile the contracts and start up ganache only once and then tasit-contracts and tasit-act use the same singleton?

But then if a dev wants to work only on `tasit-contracts` or only on `tasit-act` (or `tasit-account` if the tests there continue to use a test blockchain), then the test scripts in those packages would also use lerna to start ganache from `tasit-contracts`?

Reply:
Yes. I've taught to pull up ganache. That way it'll be started from root repo and it'll no more live in tasit-contracts. Same as working now, one test script in mono repo will call and will run:
1) start ganache (TasitSDK);
2) compile and migrate contracts by truffle (tasit-contracts);
3) test all packages;
4) stop ganache.

We can create specific scripts to test packages individually (e.g., test:tasit-account) that will run just what is necessary (in this case, bypass truffle if tasit-account there is no contract interaction and run just tasit-account test suites).

Is it make sense?

Reply:
In your last comment, did you mean `thought` rather than `taught`?

Reply:
I'm doing exactly this right now, I'm working on `tasit-account` tests (event issue) and what I did was ran `npm run ganache:start` from root repo and then `npm run test` on `tasit-account`.

Reply:
Ah yeah, great. Thanks for the clarification. The only part we potentially would want to add is making it so that running `npm test` from within `tasit-account` would automatically call `npm run ganache:start` from the root if it wasn't already started.

Reply:
It doesn't work automatically (I've tried to run `npm run ganache:start` inside `tasit-account`). One way that seems that it will work will be if we extract ganache commands to a service-like file script (ganache.js <start|stop>) and then we can run it from sub packages using `"ganache:start": "../../ganache.js start"`.

Reply:
Ah ok cool! I don't think we need to create an issue for that for now, but we can keep it in mind if we ever experience that as a pain point.